protobuf/.travis.yml

```diff
@@ -0,0 +1,18 @@
+os: linux
```

Comment:
Travis needed? 😕

Reply:
No, just to check if I can compile the project. I gonna remove it to make the example simple as possible.